packages/web-api/src/WebClient.ts

```diff
@@ -183,6 +183,9 @@ export class WebClient extends Methods {
       this.logger = getLogger(WebClient.loggerName, logLevel ?? LogLevel.INFO, logger);
     }
 
+    // eslint-disable-next-line no-param-reassign
+    if (!headers.Authorization) headers.Authorization = `Bearer ${this.token}`;
```

Comment:
`this.token` can be absent. For instance, some API calls (such as oauth.v2.access with code parameter, openid.connect.token with code parameter, api.test) do not require token in the authorization header. Can you update the logic to check if `this.token` exists too?